packages/odyssey-react-mui/src/labs/index.ts

```diff
@@ -17,7 +17,7 @@ export type { LocalizationProviderProps } from "@mui/x-date-pickers";
 export * from "./DatePicker";
 export * from "./datePickerTheme";
 
-export * from "./DataTable";
+// export * from "./DataTable";
 export * from "./DataTablePagination";
 export * from "./DataFilters";
```

Comment:
Are these still needed? Should they be moved out of Labs as well?

Reply:
DataFilters is going to stay in Labs temporarily, pending its own refactor. Pagination is moving to a separate component in another (not-yet-pushed) PR,